### ISSUE TITLE
config.yml could not be found with manual init

### ISSUE THREAD

**Issue by Nexuist** [netlify/netlify-cms#1908]
**Describe the bug**

Netlify CMS still attempts to load config.yml, even when manual init is set.

**To Reproduce**

Consider the following script:
```js
window.CMS_MANUAL_INIT = true;

import CMS, { init } from "netlify-cms";

init({
	config: ...
});
```
When the page is loaded, it flashes an error message (config.yml could not be loaded...) and then launches the CMS. However everything works as expected, collections are loaded properly, media can be uploaded, etc.

**Expected behavior**

If `CMS_MANUAL_INIT` is true, Netlify CMS should not attempt to load config.yml. It also appears that the module is loaded twice, as there are 2 version printouts and 2 404s.

**Screenshots**

![image](https://user-images.githubusercontent.com/1498061/48974699-3a78f400-f02d-11e8-9850-fa7404a4e471.png)

**Applicable Versions:**

 - Netlify CMS version: 2.2.0
 - Git provider: GitHub
 - OS: MacOS
 - Browser version: Chrome 70

 - Node.JS version: 10.6.0

**CMS configuration**

N/A

**Additional context**

N/A

**Comment by erquhart** [comment]
Attempting to load config with manual init isn't a bug, it's intentional. #1866 has been raised to provide an opt-out.

Regarding everything else you're seeing, Netlify CMS doesn't load itself, you'll need to check through your source and build setup to determine why the script is being loaded twice.

**Comment by HeyITGuyFixIt** [comment]
Even after #2053, it seems the issue still persists.

**Comment by erezrokah** [comment]
Hi @HeyITGuyFixIt, you need to explicitly configure `load_config_file: false` for the CMS not to try and load the config yml as specified in the example code here https://www.netlifycms.org/docs/beta-features/#manual-initialization.
If that is not working can you please open a new issue with a reproduction scenario?
